lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -267,110 +263,93 @@ describe('GatherRunner', function() {
     test('works when running on desktop device', DESKTOP_UA, 'desktop');
   });
 
-  it('sets up the driver to begin emulation when all emulation flags are undefined', () => {
-    const tests = {
-      calledDeviceEmulation: false,
-      calledNetworkEmulation: false,
-      calledCpuEmulation: false,
-    };
-    const createEmulationCheck = variable => (arg) => {
-      tests[variable] = arg;
-
-      return true;
-    };
-    const driver = getMockedEmulationDriver(
-      createEmulationCheck('calledDeviceEmulation'),
-      createEmulationCheck('calledNetworkEmulation'),
-      createEmulationCheck('calledCpuEmulation')
-    );
-
-    return GatherRunner.setupDriver(driver, {
+  it('sets up the driver to begin emulation when all flags are undefined', async () => {
+    await GatherRunner.setupDriver(driver, {
       settings: {
         emulatedFormFactor: 'mobile',
         internalDisableDeviceScreenEmulation: false,
       },
-    }).then(_ => {
-      assert.ok(tests.calledDeviceEmulation, 'did not call device emulation');
-      assert.deepEqual(tests.calledNetworkEmulation, {
-        latency: 0, downloadThroughput: 0, uploadThroughput: 0, offline: false,
-      });
-      assert.ok(!tests.calledCpuEmulation, 'called cpu emulation');
     });
+
+    connectionStub.sendCommand.findInvocation('Emulation.setDeviceMetricsOverride');
```

Comment:
Put an expectation around this?
```javascript
expect(connectionStub.sendCommand.findInvocation('Emulation.setDeviceMetricsOverride')).toBeDefined()
```

Reply:
shouldn't be necessary, `findInvocation` does the assertion already

Reply:
I don't love the API for it, `findInvocation` doesn't really imply that it is asserting behind the scenes. Seems like there should be two functions `findInvocation` and `expectInvocationExists`, or expect out here. But @connorjclark mentioned that this is used a lot, so might be a separate large scale change.